CMakeLists.txt

```diff
@@ -397,54 +397,93 @@ if (LLAMA_HIPBLAS)
     endif()
 endif()
 
+function(get_flags ccid ccver)
+    set(c_flags "")
+    set(cxx_flags "")
+
+    if (ccid MATCHES "Clang")
+        set(c_flags   -Wunreachable-code-break -Wunreachable-code-return)
+        set(cxx_flags -Wunreachable-code-break -Wunreachable-code-return -Wmissing-prototypes -Wextra-semi)
```

Comment:
These should be capitalized: `C_FLAGS` and `CXX_FLAGS`

We are not very consistent with this convention in the CMakeLists.txt files, but in general, variables should be capitalized.